gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -79,6 +79,14 @@
      * "dimensions = (string) dim1:dim2:dim3:dim4, dim1:dim2:dim3:dim4"
      */
 
+/**
+ * @brief Caps string for other/tensors with the number of tensors
+ */
+#define GST_TENSORS_CAP_WITH_NUM(num) \
+    "other/tensors, " \
+    "num_tensors = " num ", " \
```

Comment:
is this safe? ("..." num "...")

Reply:
I updated the description for this macro, and changed default caps string for tensors using this. Please see the changes again.